src/components/ProtectedRoute/ProtectedRoute.tsx

```diff
@@ -0,0 +1,15 @@
+import { Navigate } from 'react-router-dom';
+
+import { ROUTES } from '../../routes/Routes';
+
+interface Props {
+    children: JSX.Element;
+}
+
+export const ProtectedRoute = ({ children }: Props): JSX.Element => {
```

Comment:
you don't need to declare the return type

Reply:
Done